iranian_cities/mixins/base_location.py

```diff
@@ -0,0 +1,31 @@
+from django.db.models import Model, CharField, BigIntegerField
+from django.utils.translation import gettext_lazy as _
+
+
+class BaseLocation(Model):
+    """
+    Abstract base model for location entities.
+
+    Attributes:
+        name (str): The name of the location.
+        code (int): The unique code for the location.
```

Comment:
On model docstr we dont need attribute explanation. Because we explain each field with two powerful attributes of django. So, remove attribute explanation.

Inside docstr of models we just explain role of entity inside app context. Clarify concept of the model. Just this not more.

Reply:
@sepehr-akbarzadeh, thank you for your valuable feedback! 🎉

I've implemented the changes based on your suggestions. The docstrings for all models have been updated to focus solely on their roles within the app context, and detailed attribute explanations have been removed to utilize Django's field documentation.

Please review the updated code and let me know if there are any further adjustments needed. 🙏